TeamShiksha/logoexecutive
# Update CloudFormation Template to Add IAM Group with Read-Only Permissions

### Summary
This PR updates the existing AWS CloudFormation template by:

1. Adding an IAM group to the template.
2. Attaching a policy to the IAM group that provides read-only permissions.
3. Ensuring that IAM users belonging to this group will inherit the attached read-only permissions.

### Approach

The goal of this update is to enable a structured approach for granting read-only access to specific IAM users by organizing them into a dedicated group.

### Changes Made:

1. Added the definition for the new IAM group.
2. Created and attached a managed policy to grant read-only access.
3. Updated template documentation (if applicable) to reflect these changes.

### How to Test the Changes

1. Upload the template to AWS cloud formation.
2. Go to IAM groups and find IAMReadOnlyGroup.
3. Add an IAM user to this group enabling console access under security credentials.
4. Get the username and password for created IAM user.
5. Login again as IAM user using the credentials generated in previous step.
6. Verify all resources accessibility in IAM user's console.

### Screenshots or Recordings

https://github.com/user-attachments/assets/bc2d9e97-d01d-4b6b-9af7-dea98f35df29

## Checklist
